server/views/pages/compare/radios/hdc.njk

```diff
@@ -1,35 +1,41 @@
+{% from "./otherInput.njk" import otherInput %}
 
     {% set otherTextInput %}
-        {% include "./otherInput.njk" %}
+        {{ otherInput('HDCED-otherInput', discrepancy.causes.HDCED.other) }}
     {% endset -%}
 
     {{ govukRadios({
         classes: "govuk-radios--small",
-        name: "HDC-subType",
-
+        name: "HDCED-subType",
+        errorMessage: validationErrors | findError('HDCED-subType'),
         items: [
             {
-                value: "new-rules",
-                text: "New rules not applied"
+                value: "NEW_RULES_NOT_APPLIED",
+                text: "New rules not applied",
+                checked: discrepancy.causes.HDCED.subCategory == 'NEW_RULES_NOT_APPLIED'
             },
             {
-                value: "14-day-rule",
-                text: "14-day rule not applied "
+                value: "FOURTEEN_DAY_RULE_NOT_APPLIED",
+                text: "14-day rule not applied",
+                checked: discrepancy.causes.HDCED.subCategory == 'FOURTEEN_DAY_RULE_NOT_APPLIED'
             },
             {
-                value: "ineligible",
-                text: "Ineligible for HDCED"
+                value: "INELIGIBLE_FOR_HDCED",
+                text: "Ineligible for HDCED",
```

Comment:
Picky comment that i've raised in the JIRA, that this possibly should be `Ineligible for HDC` as the name of the scheme and not the date.